docs/manual/intro-to-boa.md

```diff
@@ -131,6 +131,32 @@ bytes.fromhex(foobar.toString('hex'));
 // "b'foobar'"
 ```
 
+### Use of Generator
```

Comment:
This should be prepended before https://github.com/alibaba/pipcook/blob/49b821201e61fb17dc63cef31806bec675b531c5/docs/manual/intro-to-boa.md#python-functions-in-worker_threads.